docs/_documentations/mdt-vsc-commands-project.md

```diff
@@ -23,20 +23,23 @@ Right-click a project in the **Codewind view** to see most project commands. All
 
-- **Open folder as workspace**: Open the project as your VS Code workspace folder. This command is useful if you want to work on just one project at a time per window. If you want the folder to open in a new window, set *window.openFoldersInNewWindow* to **true**.
+- **Open Container Shell**: Open a shell, either *bash* or *sh*, in the project application container with *docker exec*. The project must have a container running. The VS Code integrated terminal needs access to your *PATH* environment variable so that it can run the *docker* command.
 
-- **Show project overview**: Open an editor tab that displays all of the Codewind information for the project. From this page, you can build, disable, or delete the project, and you can change the project auto build setting. This page is the only place in the extension where you can delete a project. If you delete a project, you remove it from both Codewind and from your file system.
+- **Open Application Monitor**: Open the Codewind **Application Monitor** page for this project in the system browser.
 
-- **Build**: Manually request an application build for this project. If the project has auto build enabled, this command should not be necessary because builds are triggered automatically with a code change. This command is also available in the project info page.
-This command is equivalent to clicking the **Build** button in the Codewind **Overview** page.
+- **Open Performance Dashboard**: Opens the performance dashboard in the browser. This action is available only when the application is running or debugging.
 
-- **Toggle auto build**: Enable or disable auto build for the project. This command is also available in the project info page. This command is equivalent to clicking the **Auto Build** toggle in the Codewind **Overview** page.
+- **Add Project to Workspace**: Add this project's folder on your local disk to your VS Code workspace.
 
-- **Open container shell**: Open a shell, either *bash* or *sh*, in the project application container with *docker exec*. The project must have a container running. The VS Code integrated terminal needs access to your *PATH* environment variable so that it can run the *docker* command.
+- **Build**: Manually request an application build for this project. If the project has auto build enabled, this command should not be necessary because builds are triggered automatically with a code change. This command is also available in the project i page. This command is equivalent to clicking the **Build** button in the Codewind **Overview** page.
 
-- **Enable or disable project**: Enable or disable the project. This command is also available in the project info page.<br>
+- **Toggle auto build**: Enable or disable auto build for the project. This command is also available in the project overview. This command is equivalent to clicking the **Auto Build** toggle in the Codewind **Overview** page.
```

Comment:
Remove 

This command is equivalent to clicking the **Auto Build** toggle in the Codewind **Overview** page.

It has the same info as the same as the previous sentence

Reply:
Removed.